tsconfig.json

```diff
@@ -21,13 +21,19 @@
     "noEmit": true,
     "noFallthroughCasesInSwitch": true,
     "skipLibCheck": true,
-    "strict": true
+    "strict": true,
```

Comment:
maybe a good idea to update the `tsconfig.json` and other template files, to make sure that everything still works with TypeScript / ESLint / Stylelint in all files